sdk/python/requirements-ci.txt

```diff
@@ -28,4 +28,11 @@ confluent_kafka
 google
 pandavro==1.5.*
 kafka-python==1.*
-tabulate==0.8.*
\ No newline at end of file
+tabulate==0.8.*
+isort
+grpcio-tools
+mypy
+mypy-protobuf
+pre-commit
+flake8
+black
```

Comment:
would propose to set the version for the linters like grpcio-tools, mypy, black, flake8 etc.
It would be awesome when we can be sure on same version when developing

Rest is LGTM :) The import structure and linter, format would be really helpful for anyone developing next

Reply:
I agree, but we also dont want to be too restrictive. If we force everyone to confirm to our versions then its also quite high friction. Only `protoc` was a problem, and with the addition of this PR we won't be version controlling proto generated Python code, so the problem kind of goes away.

I think we can pin the versions once it becomes a problem

Reply:
Got it

Reply:
The "compatible" specifier might be a reasonable compromise, e.g. approximately expecting Semantic Versioning if a tool doesn't break behavior within a major version series:

    flake8 ~= 3.0